smsjwplatform/jwplatform.py

```diff
@@ -98,6 +98,28 @@ def key_for_media_id(media_id, preferred_media_type='video', client=None):
     return video_resource['key']
 
 
+def get_acl(key, client=None):
+    """
+
+    :param key: JWPlatform key for the media.
+    :param client: (options) an authenticated JWPlatform client as returned by
+        :py:func:`.get_jwplatform_client`. If ``None``, call :py:func:`.get_jwplatform_client`.
+    """
+    client = client if client is not None else get_jwplatform_client()
+    try:
+        video = client.videos.show(video_key=key)
+        field = video.get('video', {}).get('custom', {}).get('sms_acl', None)
```

Comment:
Do you know if there's a neat elvis operator equivalent for Python? As we start to use more of the jwplatform API, I suspect we could make use of it.